src/autopas/selectors/AutoTuner.h

```diff
@@ -343,59 +391,53 @@ bool AutoTuner<Particle, ParticleCell>::tune(PairwiseFunctor &pairwiseFunctor) {
     _numSamples = 0;
   } else {  // enough samples -> next config
     ++_currentConfig;
-    _numSamples = 0;
+    if (_currentConfig != _allowedConfigurations.end()) {
```

Comment:
probs not needed, as changing _numSamples does not matter (see below)

Reply:
removed